Move group and schedule IDs to environment

jacksonj04 opened the pull request:
Allows for groups and schedules to be switched (eg in testing) without modifying the underlying code.

These IDs are not secrets, and are used as identifiers in testing (hence their inclusion in the build and run workflows).

jacksonj04 committed:
Move group and schedule IDs to environment

Allows for groups and schedules to be switched (eg in testing) without
modifying the underlying code.

These IDs are not secrets, and are used as identifiers in testing (hence
their inclusion in the build and run workflows).

CristinaRO commented on lib/huntaway.rb:
```diff
@@ -6,8 +6,8 @@
 class Huntaway
-  FIRST_LINE_DEV_SUPPORT_GROUP_ID = 21306177
-  OPSGENIE_SCHEDULE_ID = "e71d500f-896a-4b28-8b08-3bfe56e1ed76"
+  FIRST_LINE_DEV_SUPPORT_GROUP_ID = ENV["FIRST_LINE_DEV_SUPPORT_GROUP_ID"].to_i
```
Should we use `fetch` here, so we know if we forgot to configure the env instead of getting some more confusing errors down the line with the group ID being 0?

jacksonj04 committed:
Environment variables now use fetch()

This will cause an exception if a variable is not set, rather than introducing an unknown state
